README.md

```diff
@@ -10,9 +10,7 @@ You would also need to download and set up ganache gui
 
 2. `cd ganache`
 
-3. `npm init`
-
-4. `npm start`
+3. `npm install && npm start`
```

Comment:
Having these two commands as separate steps may be better in the cases when they would fail. Then it would be easier to determine which step caused the problem.